### DISCUSSION TITLE
Running Podman create to create container in private folder set by $TMPDIR failed

### DISCUSSION THREAD

**Discussion by baopipi0707** [containers/podman#17668] [Q&A]
Running Podman create to create container in private folder set by $TMPDIR failed, I receive the error as below, I set TMPDIR as /export/log/3295/.

Error: error creating container storage: container not known
Error: error mounting storage for container ccfdf609af74392a148b73a00b65da07290611f7069ae53b2f6ee1a5e36581e0: stat /export/log/3295/1253//share/containers/storage/overlay/9ee3c79ec1b78d991d1717ec89605eb02fec0b95a1e4f502a0adc01784a35fac: no such file or directory

**Comment by baude** [comment]
this is not nearly enough information to make any sort of educated advice.  can you provide podman --info  and add --log-level=debug to the podman command

**Reply by baopipi0707** [reply]
The podman info is:

host:
  arch: amd64
  buildahVersion: 1.24.6
  cgroupControllers: []
  cgroupManager: cgroupfs
  cgroupVersion: v1
  conmon:
    package: Unknown
    path: /export/home/0248/.local/container/bin/conmon
    version: 'conmon version 2.1.0, commit: unknown'
  cpus: 64
  distribution:
    distribution: '"centos"'
    version: "7"
  eventLogger: file
  hostname: f0709
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1254
      size: 1
    - container_id: 1
      host_id: 9055072
      size: 35536
    uidmap:
    - container_id: 0
      host_id: 1253
      size: 1
    - container_id: 1
      host_id: 9055072
      size: 35536
  kernel: 3.10.0-1160.25.1.el7.x86_64
  linkmode: dynamic
  logDriver: k8s-file
  memFree: 512300802048
  memTotal: 540300972032
  networkBackend: cni
  ociRuntime:
    name: runc
    package: Unknown
    path: /export/home/0248/.local/container/sbin/runc
    version: |-
      runc version 1.1.0
      spec: 1.0.2-dev
      go: go1.16.13
      libseccomp: 2.3.1
  os: linux
  remoteSocket:
    path:/lodisk/9636/podman-run-1253/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_AUDIT_WRITE,CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_MKNOD,CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: ""
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /export/home/0248/.local/container/bin/slirp4netns
    package: Unknown
    version: |-
      slirp4netns version 1.1.12
      commit: 7a104a101aa3278a2152351a082a6df71f57c9a3
      libslirp: 4.6.1.55-888d
      SLIRP_CONFIG_VERSION_MAX: 4
      libseccomp: 2.3.1
  swapFree: 0
  swapTotal: 0
  uptime: 25h 24m 42.38s (Approximately 1.04 days)
plugins:
  log:
  - k8s-file
  - none
  - none
  - passthrough
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
registries: {}
store:
  configFile: /export/home/0248/.local/container/config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions:
    overlay.imagestore: /export/home/0248/.local/share/containers/storage
    overlay.mount_program:
      Executable: /export/home/0248/.local/container/bin/fuse-overlayfs
      Package: Unknown
      Version: |-
        fusermount3 version: 3.10.4
        fuse-overlayfs: version 1.8.2
        FUSE library version 3.10.3
        using FUSE kernel interface version 7.31
    overlay.mountopt: nodev,fsync=0
    overlay.skip_mount_home: "true"
  graphRoot:/lodisk/9636/1253/share/containers/storage
  graphStatus:
    Backing Filesystem: tmpfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  imageCopyTmpDir:/lodisk/9636
  imageStore:
    number: 4
  runRoot: /export/home/0248/rundir/containers
  volumePath:/lodisk/9636/1253/share/containers/storage/volumes
version:
  APIVersion: 4.0.2
  Built: 1677210116
  BuiltTime: Fri Feb 24 11:41:56 2023
  GitCommit: d54ecc8c3b635ee79d60a6eb4b04bd2a046f60f0-dirty
  GoVersion: go1.17.7
  OsArch: linux/amd64
  Version: 4.0.2
